uk.co.compucorp.civicrm.hrleaveandabsences/scss/components/_manage-leave-requests.scss

```diff
@@ -15,6 +15,8 @@
         padding-bottom: 20px;
       }
     }
+
+    @include uniform-padding(20px);
```

Comment:
I would put it on top along with other properties like `padding-top: 15px;` otherwise it is a bit difficult to read what is the styling for `.chr_manage_leave_requests__sub-header`. If you agree, please apply to other places as well.